apps/system/utils/utils_cpuload.c

```diff
@@ -223,19 +239,21 @@ static void cpuload_print_normal(void)
 	printf("PID | Pri |");
 
 #ifdef CONFIG_SCHED_MULTI_CPULOAD
-	for (int i = PROC_STAT_CPULOAD_SHORT; i <= PROC_STAT_CPULOAD_LONG; i++) {
+	for (int i = PROC_STAT_CPULOAD_SHORT; i <= PROC_STAT_CPULOAD_LONG; i++)
 #endif
+	{
 #ifdef CONFIG_SMP
 		for (int j = 0; j < CONFIG_SMP_NCPUS; j++) {
-			printf("  CPU%d |", j);
+			if (cpu < CONFIG_SMP_NCPUS) {
+				if (cpu == j) {
+					printf("  CPU%d |", j);
+				}
+			} else {
+				printf("  CPU%d |", j);
+			}
 		}
 #endif
-#ifndef CONFIG_SCHED_MULTI_CPULOAD
-		printf("  Avg  |");
-#else
-		printf("  Avg%d |", i - PROC_STAT_CPULOAD_SHORT + 1);
```

Comment:
We dont want an extra column of average of two core's cpuload?

Reply:
No. I feel the avg load is misleading in smp case. Let us remove it.